<ui version="4.0" >
 <class>ChartView</class>
 <widget class="QMainWindow" name="ChartView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ChartView</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="Icons/icons.qrc" >:/Icons/help.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="tableGroup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Table/Stats Views</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QFrame" name="tableFrame0" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="tableFrame1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="tableFrame2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="tableFrame3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="tableFrame4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Chart Views</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>2</number>
        </property>
        <property name="topMargin" >
         <number>2</number>
        </property>
        <property name="rightMargin" >
         <number>2</number>
        </property>
        <property name="bottomMargin" >
         <number>2</number>
        </property>
        <property name="horizontalSpacing" >
         <number>2</number>
        </property>
        <item row="0" column="0" >
         <widget class="QFrame" name="barChartFrame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QFrame" name="lineChartFrame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QFrame" name="stackedChartFrame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QFrame" name="boxChartFrame" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpenDatabase" />
    <addaction name="actionSave" />
    <addaction name="separator" />
    <addaction name="actionPrint" />
    <addaction name="actionHelp" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDatabase" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionPrint" />
   <addaction name="actionHelp" />
  </widget>
  <action name="actionOpenDatabase" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Icons/icons.qrc" >:/Icons/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open Database...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="icon" >
    <iconset resource="Icons/icons.qrc" >:/Icons/print.png</iconset>
   </property>
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="Icons/icons.qrc" >:/Icons/help.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="Icons/icons.qrc" >:/Icons/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
